api/references/when-clause-contexts.md

```diff
@@ -16,37 +16,130 @@ For example, VS Code uses when clauses to enable or disable command keybindings,
                    "when": "debuggersAvailable && !inDebugMode" },
 ```
 
-Above the built-in **Start Debugging** command has the keyboard shortcut `kb(workbench.action.debug.start)`, which is only enabled when there is an appropriate debugger available (context `debuggersAvailable` is true) and the editor isn't in debug mode (context `inDebugMode` is false).
+Above, the built-in **Start Debugging** command has the keyboard shortcut `kb(workbench.action.debug.start)`, which is only enabled when there is an appropriate debugger available (context key `debuggersAvailable` is true) and the editor isn't in debug mode (context key `inDebugMode` is false).
 
-## Conditional operators
+## Composing when-clauses with operators
```

Comment:
Changing this H2 header will break any header links. We can update our links in vscode-docs such as in https://code.visualstudio.com/docs/getstarted/keybindings#_conditional-operators but this will break any external links or bookmarks. 
Are you sure it is necessary to update this H2 header?

Reply:
Good point. How about we separate the exact header from the link target using this technique - https://developers.google.com/style/headings-targets#add-a-custom-anchor (the "Recommended" one) ?

Reply:
We try to keep to strict Markdown syntax and only embed HTML tags in rare cases.
Also not sure how this would work with Matt's Markdown header link verification, which I think would still flag that link as broken